File: Speciality.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int a,b,c;
        scanf("%d%d%d",&a,&b,&c);
        if(a>b&&a>c)
        printf("Setter\n");
        else if(b>c&&b>a)
        printf("Tester\n");
        else
        printf("Editorialist\n");
    }
}

File: Speciality.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int a=sc.nextInt();
            int b=sc.nextInt();
            int c=sc.nextInt();
            if(a>b&&a>c)
            System.out.println("Setter");
            else if(b>a&&b>c)
            System.out.println("Tester");
            else
            System.out.println("Editorialist");
        }
    }
}